docs/architecture.md

```diff
@@ -391,6 +391,34 @@ type ValidatorAPI interface {
 }
 ```
 
+### Signer
+The signer provides support the alternative _Remote Signature_ architecture
+which authorises charon to request adhoc signatures from a [remote signer instance](https://lighthouse-book.sigmaprime.io/validator-web3signer.html).
+In the _middleware_ architecture charon cannot initiate signatures itself and has to
+wait for the VC to submit signatures.
+
+Duties originating in the `scheduler` (`DutyAttester`, `DutyProposer`) are not significantly affected by this change in architecture.
+Instead of waiting for the `validatorapi` to submit signatures, these duties directly request
+signatures from the remote signer instance. The flow is otherwise unaffected.
+
+Duties originating in the `validatorapi` (`DutyRandao`, `DutyAggregator`) has to refactored to
+originate in the `scheduler`, since charon is in full control of the duties in this architecture.
```

Comment:
how would that look like? (`DutyRandao` and `DutyAggregator` originating from scheduler rather than vapi)

Reply:
Following the same pattern as DutyAttester and DutyProposer, Scheduler will just trigger the duty, fetcher will do nothing, since those duties just sign slot and epochs, then consensus, then the rest of the flow. So nothing too complex.